prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,10 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+var firstElement = languages[0];
+console.log('the first element of the array is', firstElement);
+var thirdElement = languages[2];
+console.log('the 3rd element of the array is', thirdElement);
+var length = languages.length;
+console.log('the length of the array is', length);
+var lastElement = languages[5];
```

Comment:
Hi Brighitte, you can compute the last index of the array without knowing its length in advance.

```js
var lastIndex = languages.length - 1;
var lastElement = languages[lastIndex];
```

Reply:
Ahh, ok. Step 7 wanted me to use two variables.
I was following instructions too literally-- it only told me to declare the one variable, so I stayed in that constraint lol

![prep-javascript-arrays_fix](https://user-images.githubusercontent.com/42279790/135187300-51e21991-0676-47f7-a68e-170530016c43.png)